pkgs/development/python-modules/rich/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27, typing-extensions, pygments, recommonmark, colorama }:
+
+buildPythonPackage rec {
+  pname = "rich";
+  version = "8.0.0";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15yz7rcn5cjv233xcvka9vd573pwafl31s6z9ni54r8y4k926l0v";
+  };
+
+  propagatedBuildInputs = [ typing-extensions pygments recommonmark colorama ];
+
+  doCheck = false;
```

Comment:
If the pypi package does not contain the tests please switch to the GitHub sources.

Reply:
Switched to GitHub sources. I had to patch `setup.py` though, see 9651c26b434f047cc4b7d3764ae0591ceee2b902.